content/sensu-core/0.29/guides/troubleshooting-sensu.md

```diff
@@ -0,0 +1,322 @@
+
+{{< highlight shell >}}
+Listing vhosts ...
+/
+/sensu{{< /highlight >}}
+
+_NOTE: The `/` in front the `sensu` vhost is required. If you're missing the slash, but have configured Sensu to use the vhost `/sensu`, you will see an error._
+
+If your vhost output doesn't look like the output above, create the vhost:
+
+{{< highlight shell >}}
+sudo rabbitmqctl add_vhost /sensu
+{{< /highlight >}}
+
+- Ensure that the `sensu` user is present:
+
+{{< highlight shell >}}
+sudo rabbitmqctl list_users{{< /highlight >}}
+
+This should give you output that looks like:
+
+{{< highlight shell >}}
+Listing users ...
+sensu   []
+guest   [administrator]{{< /highlight >}}
+
+If the user isn't present, add the user and the password for the user:
+
+{{< highlight shell >}}
+sudo rabbitmqctl add_user sensu secret
+{{< /highlight >}}
+
+_NOTE: If the user is present, and the password needs to be reset, you can reset it by using `sudo rabbitmqctl change_password sensu secret`_
+
+- Ensure that the `sensu` user has the correct permissions for the vhost:
+
+{{< highlight shell >}}
+sudo rabbitmqctl list_permissions -p /sensu{{< /highlight >}}
+
+You should see output that looks like the following:
+
+{{< highlight shell >}}
+Listing permissions for vhost "/sensu" ...
+sensu   .*      .*      .*{{< /highlight >}}
+
+If the permissions are not correct, you can set them via:
+
+{{< highlight shell >}}
+sudo rabbitmqctl set_permissions -p /sensu sensu ".*" ".*" ".*"
+{{< /highlight >}}
+
+Once we've ensured that our credentials are correct, we can see that RabbitMQ starts showing connections being accepted again:
+
+{{< highlight shell >}}
+tail -f /var/log/rabbitmq/rabbit\@sensu.log
+2018-06-26 01:28:35.191 [info] <0.642.0> accepting AMQP connection <0.642.0> (192.168.1.3:44816 -> 192.168.1.2:5671)
+2018-06-26 01:28:35.194 [info] <0.642.0> connection <0.642.0> (192.168.1.3:44816 -> 192.168.1.2:5671): user 'sensu' authenticated and granted access to vhost '/sensu'{{< /highlight >}}
+
+_WARNING: The credentials in this guide shouldn't be used in any production environment. If you're curious about how to better secure RabbitMQ, see our [Securing RabbitMQ Guide][5]._
+
+### SSL
+
+SSL issues are one of the more difficult ones to troubleshoot inside of Sensu. What lends to this difficulty is the way that AMQP (the protocol used by RabbitMQ) handles SSL failures, primarily in that the failure seen something like an unsupported Erlang/RabbitMQ combination is indistinguishable from an actual authentication issue.
+
+If you've already gone through the steps in the previous section to confirm that your Sensu instance is using the correct credentials to connect to your RabbitMQ instance, then you'll want to proceed through this part of the guide to rule out any issues with SSL.
+
+_PRO TIP: For troubleshooting SSL issues, the openssl tool provides a wealth of troubleshooting capabilities. To see what is possible with the tool, take a look at this [handy cheat sheet][6]._
```

Comment:
Not sure if `PRO TIP` was meant to be all bold, but it renders with only `PRO` bold.

Reply:
And sweet cheat sheet!

Reply:
Opened an issue for only **PRO** being bold. https://github.com/sensu/sensu-docs/issues/536